node/core/backing/src/lib.rs

```diff
@@ -1535,8 +1535,18 @@ async fn import_statement<Context>(
 
 	let stmt = primitive_statement_to_table(statement);
 
-	let summary = rp_state.table.import_statement(&rp_state.table_context, stmt);
+	rp_state.table.import_statement(&rp_state.table_context, stmt);
+}
 
+/// If an import summary is passed in, distribute it to the necessary subsystems. Also check for any
+/// new misbehaviors and issue necessary messages.
+#[overseer::contextbounds(CandidateBacking, prefix = self::overseer)]
+async fn distribute_statement<Context>(
```

Comment:
`distribute_statement` is not a good name for this, because distributing a statement is actually done with `StatementDistributionMessage::Share` elsewhere.

This is actually handling a summary and dispatching `Backed` notifications and misbehaviors as a result of importing a statement. Something like `post_statement_import_actions`, specific name is up to you.